sources/packages/backend/apps/api/src/services/application-offering-change-request/application-offering-change-request.service.ts

```diff
@@ -100,4 +103,73 @@ export class ApplicationOfferingChangeRequestService {
       count,
     };
   }
+
+  /**
+   * Gets application offering request change list.
+   * @param locationId location id.
+   * @param paginationOptions options to execute the pagination.
+   * @param statuses list of status that need to be included in the query.
+   * @returns list of requested application offering changes.
+   */
+  async getRequestedSummary(
```

Comment:
IMO the method name `getRequestedSummary` is not reflecting what it does.
`getApplicationOfferingChangeRequests` or `getApplicationOfferingChangeRequestSummary` could be better.

Reply:
@dheepak-aot I see the point about having `ApplicationOfferingChange` included in the method name but I am not sure if we need it since it is a dedicated service where the service name itself is already pretty explanatory.
That said, I would prefer something like `getSummaryByStatus` or similar.

Reply:
this method is in `application-offering-change-request.service.ts` file, so, I thought instead of giving a lengthly name, this makes sense. let me know devs' thoughts.

Reply:
I agree with Ann.

Reply:
@ann-aot `getSummaryByStatus` suggested by @andrewsignori-aot seems fine to me.